v2.0/sql-faqs.md

```diff
@@ -60,7 +60,25 @@ If you'd like to tell the query planner which index to use, you can do so via so
 
 ## How do I log SQL queries?
 
-For production clusters, the best way to log queries is to turn on the [cluster-wide setting](cluster-settings.html) `sql.trace.log_statement_execute`:
+There are several ways to log SQL queries.  The type of logging you use will depend on your requirements.
+
+- For per-table audit logs, turn on [SQL audit logs](#sql-audit-logs).
+- For system troubleshooting and performance optimization, turn on [cluster-wide execution logs](#cluster-wide-execution-logs).
+- For local testing, turn on [per-node execution logs](#per-node-execution-logs).
+
+### SQL audit logs
+
+{% include experimental-warning.md %}
+
+SQL audit logging is useful if you want to log all queries that are run against a table containing personally identifiable information (PII).
```

Comment:
Audit logging can also be useful for other reasons than recording access to PII.

The proper motivation is "SQL audit logging is useful if you want to log all queries that are run against specific tables."